internal/statestore/testing/fake_redis.go

```diff
@@ -45,6 +45,7 @@ func New(t *testing.T, cfg config.Mutable) func() {
 	cfg.Set("redis.pool.maxActive", PoolMaxActive)
 	cfg.Set("redis.pool.idleTimeout", PoolIdleTimeout)
 	cfg.Set("redis.pool.healthCheckTimeout", PoolHealthCheckTimeout)
+	cfg.Set("backfillLockTimeout", "5m")
```

Comment:
Hugo could you set 1m everywhere please?

Reply:
oh i missed this one, fixing it